composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "license": "GPL-3.0+",
     "require": {
-        "php": ">=5.4.0",
-        "ext-bcmath": "*"
+        "php": ">=5.4.0"
     },
     "require-dev": {
+        "ext-bcmath":                               "*",
```

Comment:
`require-dev` lists packages required for developing this package, or running tests, etc., and `require`list packages required by `src/`, where Iban & BBan validators stand.

Until you/someone find a replacement to `bcmod` function, used in both validators source code, you cannot remove this from here.